modules/HapticGlove_module/src/GloveWearable.cpp

```diff
@@ -334,6 +334,7 @@ bool GloveWearableImpl::getFingertipPoseValues(Eigen::MatrixXd& values)
 
 bool GloveWearableImpl::setFingertipForceFeedbackValues(const std::vector<int>& values)
 {
+    return true;
```

Comment:
Can we use a parameter instead of exiting prematurely? @RiccardoGrieco @lrapetti do you remember why this was needed?

Reply:
We used this premature return as an easy fix for not actuating the fingertip since (as far as I remember) we decided to use only the palm vibration in the end.

Of course using a parameter would be the proper thing to do. Probably we went for this "solution" due to time constraints.